.github/workflows/reset-playground.yaml

```
@@ -0,0 +1,24 @@
+name: Reset running Meshery playground
+on:
+   workflow_dispatch:
+   schedule:
+     - cron:  '0 0 * * *'  # Run the job every day at midnight
+
+jobs:
+   reset:
+     runs-on: ubuntu-latest 
```

```suggestion
     name: Cluster Reset
      if: github.repository == 'meshery/meshery'
     runs-on: ubuntu-latest 
```

Only allow to be run from this repo.